Python-Home-Challenges/Python-home-challenge#01-MaorWeiss.py

```diff
@@ -0,0 +1,27 @@
+# HW03 - Open a text file and returns the most recurring word in that file.
+
+fp = open('/file.txt', 'r')
+# Open file #
+f = fp.readline()
+# Read file #
+split_words_from_file = f.split()
+
+the_most_recurring_word = ''
+biggest_counting = 0
+count = 0
+
+copy_list = split_words_from_file
+for list_arg in copy_list:
+    for word in split_words_from_file:
+        if list_arg == word:
+            count += 1
+    if count > biggest_counting:
+        the_most_recurring_word = list_arg
+        biggest_counting = count
+        count = 0
+    else:
+        count = 0
+
+print 'The word "{}" which has appeared {} times is the most recurring word in the file.'.format(the_most_recurring_word, biggest_counting)
```

Comment:
syntax error, missing '('
also, using f' string is considered as best practice for python 3